src/main/java/no/nav/familie/inntektsmelding/forespørsel/modell/ForespørselEntitet.java

```diff
@@ -31,50 +37,31 @@ public class ForespørselEntitet {
     @Column(name = "uuid", nullable = false, updatable = false)
     private UUID uuid;
 
-    @Column(name = "sak_id")
-    private String sakId;
+    @JsonBackReference
```

Comment:
Trenger vi denne? Har ikke sett denne før hva brukes det til?

Reply:
Nei, den kan fjernes. Det er for å unngå uendelig loop av serialisering ved sirkulere avhengigheter, men eg fjernet den avhengigheten og glømte å ta bort denne annotasjonen.